docs/documents.mdx

```diff
@@ -786,145 +789,16 @@ var nitric = query.Fetch();
 
 #### Subcollection Queries
 
-Subcollection queries can be created either:
-
-1. under a collection's subcollection `collection.collection.query`, or
-2. under a collection document's subcollection `collection.document.collection.query`
-
-In the first example we are performing a collection subcollection query where we
-are fetching all "Orders" documents for all "Customers".
-
-<CodeExamples 
-	languages={[
-	{
-		label: "Node.js",
-		value: "node"
-	},
-	{
-		label: "Python",
-		value: "python"
-	},
-	{
-		label: "Java",
-		value: "java"
-	},
-	{
-		label: "Go",
-		value: "go"
-	},
-	{
-		label: "PHP",
-		value: "php"
-	},
-	{
-		label: "C#",
-		value: "c-sharp"
-	}
-	]}
-	defaultLang="node"
->
-<CodeExample lang="node">
-
-```typescript
-import { documents } from '@nitric/sdk';
-
-const docs = documents();
-
-const query = docs.collection('Customers').collection('Orders').query();
-
-const results = await query.fetch();
-```
-
-</CodeExample>
-<CodeExample lang="python">
-
-```python
-from nitric.api import Documents
-
-docs = Documents()
-
-query = docs.collection("Customers")
-		.collection("Orders")
-		.query()
-
-results = query.fetch()
-```
-
-</CodeExample>
-
-<CodeExample lang="java">
-
-```java
-import io.nitric.api.document.Documents;
-
-var query = Documents.collection("Customers")
-		.collection("Orders")
-		.query();
-
-var results = query.fetch();
-```
-
-</CodeExample>
-
-<CodeExample lang="go">
-
-```go
-import (
-	"github.com/nitrictech/go-sdk/api/documents"
-)
-
-docs, _ := documents.New()
-
-query := docs.Collection("Customers")
-		.Collection("Orders")
-		.Query()
-
-results, err := query.Fetch()
-```
+Subcollection queries can be created under a collection document's subcollection `collection.document.collection.query`
```

Comment:
I would recommend changing the order of the subcollection queries, putting collection.document.collection query first. Then put in comments about not yet implemented for the SDK which don't support this yet.